MANIFEST.in

```diff
@@ -2,3 +2,5 @@ include COPYING
 include COPYING.LESSER
 include README.rst
 include LICENSE.txt
+prune doc
+prune tests
```

Comment:
It would be better to move this change into a separate PR because it's not related to black/isort, do you agree?
Note also that there are other files that were excluded by the sdist using the previous approach, and that would be included if we don't list them here:
```
.github/workflow/*
.gitignore
.readthedocs.yml
CHANGELOG.rst
pyproject.toml
tox.ini
```

Including:
- `CHANGELOG.rst` and `pyproject.toml` seems ok to me.
- `.github` `.gitignore` `.readthedocs.yml` seems useless because related to the CI pipelines or git.
- `tox.ini`: if we don't include `tests` and `docs`, it doesn't seem useful.

Reply:
Fair enough, reverted.